File: Division.php
<?php

$dividendo = readline("Dividendo: ");
$divisor = readline("Divisor: ");
if($dividendo % $divisor ==0){
    echo "La divion es exacta \n";
}else{
    echo"La division no es exacta\n";
}
 $cuociente = floor($dividendo / $divisor);
 $resto = $dividendo % $divisor;
 
 echo "Cuociente: $cuociente \n";
 echo "resto: $resto \n";

File: Division.py
import math

dividendo = int(input("Dividendo: "))
divisor = int(input("Divisor: "))

if dividendo % divisor == 0 :
    print("La divion es exacta \n") 
else:
    print("La division no es exacta\n")

cuociente = math.floor(dividendo / divisor)
resto = dividendo % divisor
 
print("Cuociente:", cuociente, "\n") 
print("resto:", resto, "\n") 